save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarLinks.kt

```diff
@@ -57,7 +57,7 @@ private fun topBarLinks() = FC<TopBarLinksProps> { props ->
         sequenceOf(
             TopBarLink(hrefAnchor = FrontendRoutes.DEMO.path, width = 4.rem, text = "Demo"),
             TopBarLink(hrefAnchor = "${FrontendRoutes.DEMO.path}/cpg", width = 3.rem, text = "CPG"),
-            TopBarLink(hrefAnchor = FrontendRoutes.VULNERABILITIES.path, width = 6.rem, text = "Vuln. Archive"),
```

Comment:
Our topbar becomes too huge...

Reply:
Yes. It will be necessary to think about how it can be changed.